### ISSUE TITLE
Download profile picture

### ISSUE THREAD

**Issue by GniLudio** [TheKonka/instagram-download-browser-extension#30]
It would be nice to be able to download the profile picture.

**Comment by TheKonka** [comment]
Hi, thanks for your feedback. This feature was there, but it was disabled due to an update on the Instagram website. I have updated a new version 2.1.6 to fix the issue. You can try it on Chrome now. The Firefox version is still under review.

**Comment by GniLudio** [comment]
It works, but the image isn't downloaded in high quality.
(some websites allow to download the profile picture in the original quality)

**Comment by TheKonka** [comment]
> https://instazoom.cc/en

Yes, I know that too, like this URL, but the quality of the images I tried to download is the same.
Can you provide me with a web address that download the profile picture in the original quality, and an username of Instagram?

**Comment by GniLudio** [comment]
> > https://instazoom.cc/en
> 
> Yes, I know that too, like this URL, but the quality of the images I tried to download is the same. Can you provide me with a web address that download the profile picture in the original quality, and an username of Instagram?

I don't know how they do that, but the [save-free](https://www.save-free.com/) website manages to get the url of the original profile images.

For example for the [Instagram](https://www.instagram.com/instagram) profile:
[https://www.save-free.com/cdn/https://instagram.fqro3-1.fna.fbcdn.net/v/t51.2885-19/281440578_1088265838702675_6233856337905829714_n.jpg?_nc_ht=instagram.fqro3-1.fna.fbcdn.net&_nc_cat=1&_nc_ohc=Ozy2qvrb0tsQ7kNvgFPUPRJ&edm=AEF8tYYBAAAA&ccb=7-5&oh=00_AYD8HXuMGgffG3Ovy3wMWEfZVx-SfGqbh7UBom85GxiJ9g&oe=667552D8&_nc_sid=1e20d2](https://www.save-free.com/cdn/https://instagram.fqro3-1.fna.fbcdn.net/v/t51.2885-19/281440578_1088265838702675_6233856337905829714_n.jpg?_nc_ht=instagram.fqro3-1.fna.fbcdn.net&_nc_cat=1&_nc_ohc=Ozy2qvrb0tsQ7kNvgFPUPRJ&edm=AEF8tYYBAAAA&ccb=7-5&oh=00_AYD8HXuMGgffG3Ovy3wMWEfZVx-SfGqbh7UBom85GxiJ9g&oe=667552D8&_nc_sid=1e20d2)

(just remove the `https://www.save-free.com/cdn/` from the url gives you the proper image link)

**Comment by TheKonka** [comment]
https://scontent-tpe1-1.cdninstagram.com/v/t51.2885-19/281440578_1088265838702675_6233856337905829714_n.jpg?_nc_ht=scontent-tpe1-1.cdninstagram.com&_nc_cat=1&_nc_ohc=42k2WkwePkMQ7kNvgErHaeJ&edm=AFg4Q8wBAAAA&ccb=7-5&oh=00_AYB7-e29W0Gt1PQDSRKXV8xMDfpruUQdDwGZrXmRzBvZ1g&oe=66782E18&_nc_sid=cf751b

https://github.com/TheKonka/instagram-download-browser-extension/assets/22173084/22919869-ce8d-424d-9a8d-22b2cd763b40

<img width="265" alt="image" src="https://github.com/TheKonka/instagram-download-browser-extension/assets/22173084/6746233d-e995-4447-bc91-3762fcd80e33">


<img width="265" alt="image" src="https://github.com/TheKonka/instagram-download-browser-extension/assets/22173084/575221b6-f854-40ea-a6c2-9c1f2b76bb8a">



Hello, I tried comparing the downloaded images and they are the same quality. Doesn't it look like that on your side?

**Comment by GniLudio** [comment]
If I use that, the image has low quality:
![281440578_1088265838702675_6233856337905829714_n (1)](https://github.com/TheKonka/instagram-download-browser-extension/assets/50866361/88608cca-da11-43e8-9664-0694fcb78901)

https://scontent-ham3-1.cdninstagram.com/v/t51.2885-19/281440578_1088265838702675_6233856337905829714_n.jpg?stp=dst-jpg_s150x150&_nc_ht=scontent-ham3-1.cdninstagram.com&_nc_cat=107&_nc_ohc=42k2WkwePkMQ7kNvgG83s47&edm=AEhyXUkBAAAA&ccb=7-5&oh=00_AYCusXKlyxFwpSCHyFu10NLWnSPVQmsyW5dL8r16BqEhMA&oe=6679B7D8&_nc_sid=cf751b

**Comment by TheKonka** [comment]
Ok, please try uninstalling and reinstalling the latest version 2.1.7 and try again!

**Comment by GniLudio** [comment]
> Ok, please try uninstalling and reinstalling the latest version 2.1.7 and try again!

Image quality still low after reinstalling. :(
